packages/terra-alert/src/Alert.jsx

```diff
@@ -163,7 +170,9 @@ const Alert = ({
   const alertSectionClassName = cx('section', { 'section-custom': type === AlertTypes.CUSTOM });
   const alertMessageContent = (
     <div className={alertSectionClassName}>
-      {(title || defaultTitle) && <strong className={cx('title')}>{title || defaultTitle}</strong>}
+      <span aria-hidden={type === AlertTypes.ALERT ? 'true' : undefined}>
+        {(title || defaultTitle) && <strong className={cx('title')}>{title || defaultTitle}</strong>}
```

Comment:
Is there a need to create the span element if there is no title or default title?  It appears that an empty span would be created in this scenario.

Reply:
That's a good catch, after some offline discussions however it was determined that the title should not be hidden by the screen-reader since not all screen-readers will read the Alert the same and future updates may change this. The span has been removed altogether in this commit here https://github.com/cerner/terra-core/pull/3789/commits/ad9839c15b2823dcc9dfe479826a330a41979f44

Adding explanation from Eric here: "Aria-live hidden should not be added to the alert title. The redundance of "alert alert" is an unfortunate side effect of JAWS so this change causes the title to not be read on other screen readers, and possibly future versions of JAWS if they were to ever change that. The correct implementation is to have to title read."